attribution-reporting/functions/apps/adtech.js

```diff
@@ -142,11 +142,9 @@ adtech.get('/ad-script-click-js', (req, res) => {
 /*                  Source registration (ad click or view)                    */
 /* -------------------------------------------------------------------------- */
 
-adtech.get(['/register-source-image', '/register-source-href'], (req, res) => {
-  const attributionDestination = process.env.ADVERTISER_URL
-  // For demo purposes, sourceEventId is a random ID. In a real system, this ID would be tied to a unique serving-time identifier mapped to any information an adtech provider may need
-  const sourceEventId = Math.floor(Math.random() * 1000000000000000)
-  const legacyMeasurementCookie = req.cookies['__session']
+adtech.get(
+  ['/register-source-image', '/register-source-href', '/register-source-js'],
+  (req, res) => {
```

Comment:
Shouldn't line 175 also be updated to redirect `/register-source-js`?

Reply:
Fixed 👍